### DISCUSSION TITLE
Is there an alternative solution for multiple text input?

### DISCUSSION THREAD

**Discussion by Tim-Cao** [Textualize/textual#3257] [Q&A] [answered]
Btw, TextArea (multi-line input) in roadmap, when will it be released? Is there a plan?
https://textual.textualize.io/roadmap/#widgets

**Comment by davep** [comment, marked as answer]
See https://github.com/Textualize/textual/pull/2931

**Reply by Tim-Cao** [reply]
Thanks. Look forward it
